book/src/builders.md

```diff
@@ -21,9 +21,7 @@ The beacon node and validator client each require a new flag for lighthouse to b
 ```
 lighthouse bn --builder https://mainnet-builder.test
 ```
-The `--builder` flag will cause the beacon node to query the provided URL during block production for a block
-payload with stubbed-out transactions. If this request fails, Lighthouse will fall back to the local
-execution engine and produce a block using transactions gathered and verified locally.
+The `--builder` flag will cause the beacon node to simultaneously query the provided URL and the local execution engine during block production for a block payload with stubbed-out transactions.
```

Comment:
I think this also loses information compared to what was there. It might be good to keep the second sentence (or something similar) about what happens if either fails

Reply:
Add more comment about the case of `either fails` and `both succeed`.